src/Stories.js

```diff
@@ -8,31 +7,37 @@ class Stories extends Component {
    constructor(props) {
     super(props);
     this.state = {'stories': [], 'currentVote': [] };
+    this.updateComponent = this.updateComponent.bind(this)
    }
 
    componentDidMount(){
+     this.updateComponent()
+   }
+
+   updateComponent(){
      var that = this;
      request
-      .get('http://master.bass-seahorse-cod.app.push.drieapp.co/stories?state=active')
+      .get('http://master.bass-seahorse-cod.app.push.drieapp.co/stories')
       .end(function(err, res){
        that.setState({"stories": JSON.parse(res.text)})
       })
    }
 
-    mapOpenStoryItems(){
+    render() {
       if(this.state.stories.length > 0 ){
+      var storyItems = this.state.stories.map((story) => {
         return <Story
-                  title={this.state.stories[0].name}
-                  votes={this.state.stories[0].size}
-                  url={this.state.stories[0].url}
-                  id={this.state.stories[0]._id} />
+                  key={story.id}
+                  title={story.name}
+                  votes={story.size}
+                  url={story.url}
+                  update={this.updateComponent}
+                  id={story._id} />
+      })
```

Comment:
the above looks like an improvement, but could we use something like ```<Story story={story}>```

and then the Story.js would have something like:

``` <li>Title: {this.props.story.title}</li> ```